hpx/runtime/threads/policies/scheduler_base.hpp

```diff
@@ -307,6 +315,8 @@ namespace hpx { namespace threads { namespace policies {
 
         std::atomic<std::int64_t> background_thread_count_;
 
+        std::function<void(void)> user_polling_function_;
```

Comment:
Please use `hpx::util::unique_function_nonser` for this. It's more lightweight than any other `std::function` implementations we know.